src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseDetailsEndpoint.java

```diff
@@ -112,6 +114,7 @@ public CaseDetailsEndpoint(@Qualifier(CreatorGetCaseOperation.QUALIFIER) final G
         @ApiResponse(code = 400, message = "Invalid case ID"),
         @ApiResponse(code = 404, message = "No case found for the given ID")
     })
+    @LogAudit(operationType = OperationType.VIEW_CASE)
```

Comment:
this endpoint is not used by the UI when the user is viewing a case. So the name of the operation might be a bit misleading. I wonder whether we want to introduce a separate operation name for when the case is being accessed programmatically like in this case. Rather than by the UI. I'll ask the Lee and the BAs

Reply:
Please check and let us know. Same as other comments. Will clean up and will be in v1 PR.